client/src/modules/home/shared/components/Layout.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Layout: React.FC = ({ children }) => {
```

Comment:
What is the purpose of this component?

Reply:
I was looking at how [`client/src/modules/dashboard/shared/components/Layout.tsx`](https://github.com/freeCodeCamp/chapter/blob/8f836a934ffccabb6837cd5f4989292d0687e104/client/src/modules/dashboard/shared/components/Layout.tsx) was rendered, but didn't want to have the Navbar.

Reply:
If that's the case, we can do without this component. It can be added in when it's needed, otherwise right now it's not doing anything. Why do you not want the nav rendered though?

Reply:
Approving assuming this file will become something later on.